### DISCUSSION TITLE
About the addition of new authentication methods.

### DISCUSSION THREAD

**Discussion by iliaoke** [ente-io/ente#3381] [Enhancements]
Is it possible to add a verification method for verifying steam? See other 2FA apps that support steam.

**Comment by Damariobros** [comment]
Ente already supports Steam authentication! Just type in "Steam" in the Issuer field and the secret you enter will be treated as a Steam Guard secret.

**Comment by iliaoke** [comment]
👌,thank you



---Original---
From: ***@***.***&gt;
Date: Sun, Sep 22, 2024 13:04 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ente-io/ente] About the addition of new authentication methods.(Discussion #3381)




 
Ente already supports Steam authentication! Just type in "Steam" in the Issuer field and the secret you enter will be treated as a Steam Guard secret.
